packages/extension-code-block-lowlight/package.json

```diff
@@ -27,13 +27,14 @@
   "dependencies": {
     "@tiptap/extension-code-block": "^2.0.0-beta.40",
     "@types/lowlight": "^0.0.3",
-    "prosemirror-model": "^1.16.1",
-    "prosemirror-state": "^1.3.4",
-    "prosemirror-view": "^1.23.6"
+    "lowlight": "^1.20.0",
```

Comment:
Hi @bdbch! 

I’m trying to [upgrade tiptap packages](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/88851#note_999312652) in our application, and I noticed that in the latest released version of `code-block-lowlight` declares the `lowlight` package as a dependency **and** a peerDependency: https://unpkg.com/browse/@tiptap/extension-code-block-lowlight@2.0.0-beta.72/package.json

This is not the case in the master branch: https://github.com/ueberdosis/tiptap/blob/main/packages/extension-code-block-lowlight/package.json#L25. 

What I find confusing is that part of the changes that I submitted in the [PR that moves lowlight to a peerDependency](https://github.com/ueberdosis/tiptap/pull/2625/files) are also included in the latest released version, which differs from what exists in the master branch. 

Why is the package.json file different in the master branch and the released package?